spacy/lang/hy/lex_attrs.py

```diff
@@ -28,10 +28,10 @@
     "քսան" "երեսուն",
```

Comment:
As @mahnerak pointed out at PR #5246, there probably should be a comma between these two words?

Reply:
Yes, we do need a comma there.

Reply:
@myavrum: could you add that additional edit? If you just push it to this `patch-1` branch on your personal repo, it will automatically show up in this PR.

Reply:
@svlandeg I have the change (and some other minor fixes) in `YerevaNN/spaCy`. Is it possible to pull the commit from there?

Reply:
You know what, actually, maybe it's easier if you create a follow-up PR once this is merged. You agree with all the current changes in this PR, right?

Reply:
Absolutely!

Reply:
yes